install/wipe_ncn_disks_for_reinstallation.md

```diff
@@ -95,80 +95,110 @@ wiping the disks and RAIDs.
+           66a78adf6b4c2       18b6035f5a9ce       About a minute ago   Running             spire-bundle                                        1212                6d89f7dee8ab6
+           7680e4050386d       c8344c866fa55       24 hours ago         Running             speaker                                             0                   5460d2bffb4d7
+           b6467c907f063       8e6730a2b718c       3 days ago           Running             request-ncn-join-token                              0                   a3a9ca9e1ca78
+           e8ce2d1a8379f       64d4c06dc3fb4       3 days ago           Running             istio-proxy                                         0                   6d89f7dee8ab6
+           c3d4811fc3cd0       0215a709bdd9b       3 days ago           Running             weave-npc                                    0                   f5e25c12e617e
+           ```
+
+       1. If there are any running containers from the output of the `crictl ps` command, stop them.
+
+           ```bash
+           ncn-m# crictl stop <container id from the CONTAINER column>
+           ```
+
 
 1. Delete Ceph Volumes **on storage nodes ONLY**.
 
-    For each storage node:
+    1. For each storage node:
 
-    1. Stop Ceph.
+       1. Stop Ceph.
 
-        * ***1.4 or earlier***
+           * ***Shasta 1.4 or earlier***
 
-            ```bash
-            ncn-s# systemctl stop ceph-osd.target
-            ```
+               ```bash
+               ncn-s# systemctl stop ceph-osd.target
+               ```
 
-        * ***1.5 or later***
+           * ***Shasta 1.5 or later***
 
-            ```bash
-            ncn-s# cephadm rm-cluster --fsid $(cephadm ls|jq -r '.[0].fsid') --force
-            ```
+               ```bash
+               ncn-s# cephadm rm-cluster --fsid $(cephadm ls|jq -r '.[0].fsid') --force
+               ```
 
-    1. Make sure the OSDs (if any) are not running.
+       1. Make sure the OSDs (if any) are not running.
 
-        * ***1.4 or earlier***
+           * ***1.4 or earlier***
 
-            ```bash
-            ncn-s# ps -ef|grep ceph-osd
-            ```
+               ```bash
+               ncn-s# ps -ef|grep ceph-osd
+               ```
 
-        * ***1.5 or later***
+           * ***1.5 or later***
 
-            ```bash
-            ncn-s# podman ps
-            ```
+               ```bash
+               ncn-s# podman ps
+               ```
 
-        Examine the output. There should be no running `ceph-osd` processes or containers.
+           Examine the output. There should be no running `ceph-osd` processes or containers.
 
-    1. Remove the VGs.
+       1. Remove the VGs.
 
-        ```bash
-        ncn-s# ls -1 /dev/sd* /dev/disk/by-label/*
-        ncn-s# vgremove -f -v --select 'vg_name=~ceph*'
-        ```
+           ```bash
+           ncn-s# ls -1 /dev/sd* /dev/disk/by-label/*
+           ncn-s# vgremove -f -v --select 'vg_name=~ceph*'
```

Comment:
These are cleaned up by dracut already, and are not subject to the wipe bug in MTL-1288.
This line could be removed.

For reference, the dracut code that removes these is here:
- Defined VGs to remove: https://github.com/Cray-HPE/dracut-metal-mdsquash/blob/main/90metalmdsquash/metal-md-lib.sh#L309-L310
- Code removing the VGs https://github.com/Cray-HPE/dracut-metal-mdsquash/blob/main/90metalmdsquash/metal-md-lib.sh#L325-L328

On the other hand, if this is desired still then MTL-1288 will remove these calls, @cdelatte-hpe might have an opinion here.

Reply:
Not changing this until there is more feedback.

Reply:
@haroldlongley i think we can leave this as is.

I'll pencil this into my notes for MTL-1288.

I'll leave this open for you to resolve, incase Craig wants to comment.